examples/player/simid_player.js

```diff
@@ -282,13 +288,16 @@ class SimidPlayer {
    * @return {!Object}
    */
   getNonlinearDimensions_() {
-    let newDimensions = {};
-    newDimensions.x = document.getElementById('x_val').value;
-    newDimensions.y = document.getElementById('y_val').value;
-    newDimensions.width = document.getElementById('width').value;
-    newDimensions.height = document.getElementById('height').value;
-
-    return newDimensions;
+    if(this.resizeDimensions_) {
+      return this.resizeDimensions_;
+    } else {
```

Comment:
Since you're returning up there, you don't even need to put this part in an else clause

Reply:
Fixed